src/deepsparse/yolact/schemas.py

```diff
@@ -68,8 +69,31 @@ class YOLACTInputSchema(ComputerVisionSchema, Splittable):
         "`postprocess` step (optional)",
     )
 
+    @classmethod
+    def from_files(cls, files: List[str], **kwargs) -> "YOLACTInputSchema":
```

Comment:
```suggestion
    def from_files(cls, files: Iterable[TextIO], **kwargs) -> "YOLACTInputSchema":
```

nit: I would like to rely on the behaviour rather than forcing files to be a List, also files is not always an Iterable of strs, but could also be a File Pointer, TextIO is a parent type which catches all that

Reply:
@corey-nm you down? I agree with @rahul-tuli

Reply:
Also agree!